src/pages/sighting/SightingCore.jsx

```diff
@@ -119,18 +132,29 @@ export default function SightingCore({
       /> */}
       <ConfirmDelete
         open={deleteDialogOpen}
-        onClose={() => setDeleteDialogOpen(false)}
+        onClose={() => {
+          setMessageForConfirmDelete(
+            'CONFIRM_DELETE_SIGHTING_DESCRIPTION',
+          );
+          setVulnerableIndividual(null);
```

Comment:
The message does not need to be set here. Clearing the vulnerable individual guid will cause the 'CONFIRM_DELETE_SIGHTING_DESCRIPTION' message to be used.

Reply:
Good point.